src/data/styles/header.scss

```diff
@@ -0,0 +1,385 @@
+    width: 22%;
+    display: flex;
+    justify-content: space-between;
+    letter-spacing: 0.2px;
+
+    .search {
+      i {
+        font-size: 13px;
+        margin-right: 6.5px;
+      }
+    }
+
+    .user-name {
+      i {
+        font-size: $font-sm;
+        margin-right: 7px;
+      }
+    }
+
+    .quantity {
+      color: $cobalt-blue;
+      opacity: 1;
+      position: relative;
+      font-size: 19px;
+      margin-left: 9px;
+
+      i {
+        color: $white;
+        position: absolute;
+        top: -5px;
+        left: 33%;
+        font-size: $font-xs;
+      }
+    }
+  }
+
+  .brand-info {
+    display: flex;
+    flex-direction: column;
+
+    .logo {
+      height: 26px;
+      background: url("../assets/header/logo.png") no-repeat center;
+    }
+
+    .brand-name {
+      color: $white;
+      padding-top: 10px;
+    }
+  }
+
+  &:hover {
+    color: $black;
+    background-color: $white;
+    background-image: none;
+
+    .logo {
+      background: url("../assets/header/logo-black.png") no-repeat center;
+    }
+
+    .brand-name {
+      color: $black;
+    }
+  }
+}
+
+.stories-dropdown {
+  padding: 0 22px;
+  border-top: 1px solid $light-grey;
+  display: none;
+  position: absolute;
+  top: 120px;
+  background-color: $white;
+  z-index: 9;
+
+  ul {
+    padding: 32px 63px 49px;
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    list-style: none;
+
+    a {
+      color: $grey;
+      text-align: center;
+
+      img {
+        width: 216px;
+        min-height: 16.6875rem;
+        margin-bottom: 5px;
+      }
+
+      p,
+      span {
+        line-height: 1rem;
+        letter-spacing: 0.2px;
+      }
+
+      p {
+        font-size: 0.8125rem;
+        font-weight: 600;
+        margin-top: 1rem;
+      }
+
+      span {
+        font-size: 0.6875rem;
+        font-weight: 400;
+        display: block;
+        margin-top: 0.4375rem;
+      }
+    }
+
+    .stories-info {
+      border-right: 1px solid $light-grey;
+      padding-top: 12px;
+      margin-right: 16px;
+      min-width: 16.375rem;
+      color: $grey;
+      font-size: $font-md;
+
+      div {
+        margin-bottom: 20px;
+        max-width: 13.3125rem;
+        font-size: $font-md;
+        line-height: 1.5rem;
+      }
+
+      strong {
+        margin-bottom: 0.75rem;
+        font-size: 0.8125rem;
+        font-weight: 600;
+        line-height: 1;
+
+        &:hover {
+          text-decoration: underline;
+        }
+      }
+
+      .mob-text {
+        display: none;
+      }
+    }
+
+    .dropdown-img-card {
+      grid-auto-flow: column;
+      justify-content: end;
+      margin: 0 0 0 auto;
+      padding-top: 14px;
+      padding-left: 32px;
+    }
+
+    .dropdown-mob-image {
+      display: none;
+    }
+  }
+
+  &.is-opened {
+    display: block;
+  }
+}
+
+.rotate-icon {
+  transform: rotate(-180deg);
+}
+
+.nav-white-bg {
+  background-color: $white;
+  background-image: none;
+}
+
+.sticky {
```

Comment:
To keep things clean, this and other helper classes should be moved to another scss module and include them wherever needed. Something like the `common.scss` you are using